dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/sms/listener/AggregateDataSetSMSListener.java

```diff
@@ -110,10 +110,10 @@ protected SMSResponse postProcess( IncomingSms sms, SMSSubmission submission )
         String per = subm.getPeriod();
         UID aocid = subm.getAttributeOptionCombo();
 
-        OrganisationUnit orgUnit = organisationUnitService.getOrganisationUnit( ouid.uid );
-        User user = userService.getUser( subm.getUserID().uid );
+        OrganisationUnit orgUnit = organisationUnitService.getOrganisationUnit( ouid.getUID() );
+        User user = userService.getUser( subm.getUserID().getUID() );
```

Comment:
Use `.getUid()` consistently.

Reply:
@larshelge - case of acronym PR here: https://github.com/dhis2/dhis2-core/pull/5423